content/news/2021-12-22-dvs-internals-pt2.md

```diff
@@ -239,11 +239,11 @@ The Serato Control CD has a play time of approximately 16 minutes 20 seconds, or
 At a timecode frequency of 1000 Hz (cycles per second), we have 980,000 cycles.
 Every cycle encodes one bit, therefore we need an LFSR with an output length of at least 980,000 bits before it starts to repeat.
 
-At most, an n-bit LFSR can output 2<sup>n - 1</sup> bits before it starts to repeat.
+At most, an n-bit LFSR can output 2<sup>n</sup> - 1 bits before it starts to repeat.
```

Comment:
lmao, wikipedia is also wrong on this. https://en.wikipedia.org/wiki/Linear-feedback_shift_register#Output-stream_properties

Reply:
Interesting, I think it's correct in other sections of the article. Not that this would make it any less confusing... I usually don't feel confident enough to edit math-related things on Wikipedia myself, but here it might actually make sense to correct it 😄